qemu/tests/cfg/hotplug_mem.cfg

```diff
@@ -137,8 +137,9 @@
             sub_type = boot
             reboot_method = system_reset
             sleep_before_reset = 0
+            sub_test_wait_time = 60
```

Comment:
Since we already use `sleep_before_reset = 0`,  So, 'system_reset' will be executed soon,  i think  `sub_test_wait_time = 60` is too long,  vm may already boot up absolutely.   Then hot plug memory after vm boot up is not what we want too.

Reply:
Yeah, we need to get a value which can ensure memory is hotplugged during guest booting and also check_memory() won't timeout. `sub_test_wait_time = 60` works for my guest. But I guess we might need a better approach to suit all guests. I will try your suggestion by getting a boot up time. Thanks.

Reply:
@yanan-fu `sub_test_wait_time` does not only work for guest_reboot and system_reset cases, but for other sub tests as well. Then using boot up time may not be appropriate for them. If only use it for the two cases, seems tricky.  What do you think?

Reply:
We can get boot up time, and only use it when can not get `sub_test_wait_time` from params.  Then do not set `sub_test_wait_time` for these two case is ok.

Reply:
That is not completely true, default value is `0` for all the cases.  But i do not have other good suggestion to make `hotplug` be executed in `during` absolutely.

Reply:
Only `(sub_type == boot or sub_type == reboot) and stage ==during`, to get boot up time.  Others, do nothing.

Reply:
Updated to get guest boot up time for reboot cases. Please help review, thanks.